### PR TITLE
#U15 feat: implement the expertise section for desktop breakpoints

### PR DESCRIPTION
**Pull request by Yvad60**
# PR Description

Add expertise section for desktop breakpoints

# Description of tasks that were expected to be completed

- [x] Implement expertise section for desktop breakpoints
- [x] change the expertise width on desktop breakpoints
- [x] ensure new changes doesn't break existing features

# How this maybe tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. checkout to the current branch
2. run `npm i` to install all required dependences 
3. run `npm run dev`

Provide screenshots related to your PR in case its appropriate and applicable

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: components/Sections/Expertise.jsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import expertiseList from "../../utils/expertiseData";
 export default function Expertise() {
   return (
-    <section className="float-right">
+    <section className="float-right mb-[4.8125rem] md:mb-[8.25rem] lg:mb-24">
```
Comment (chrisostome01) [inline components/Sections/Expertise.jsx]: It would be better if you add them in tailwind config

Comment (chrisostome01) [reply components/Sections/Expertise.jsx]: I mean the arbitrary values

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
Well done @Yvad60, I left some comments you may take look

File: components/Sections/Expertise.jsx (hunk 1 above)
Comment (Yvad60) [reply components/Sections/Expertise.jsx]: Okay..

**Review by chrisostome01 — APPROVED** [review]
well done 🚀

File: components/Sections/Expertise.jsx [hunk 2]
```diff
@@ -1,15 +1,15 @@
 import expertiseList from "../../utils/expertiseData";
```
Comment (serapieTuyishime) [inline components/Sections/Expertise.jsx]: this ../../ it not necessary if you remove them there will be no change

**Review by serapieTuyishime — CHANGES_REQUESTED** [review]

File: components/Sections/Expertise.jsx (hunk 2 above)
Comment (Yvad60) [reply components/Sections/Expertise.jsx]: Really? 
I didn't know about that. let me try

Comment (Yvad60) [reply components/Sections/Expertise.jsx]: It works

Comment (serapieTuyishime) [reply components/Sections/Expertise.jsx]: !

**Review by serapieTuyishime — APPROVED** [review]
🚀

**Pull request merged**